test/receiverSpec.js

```diff
@@ -518,6 +519,103 @@ describe('Receiver Module', function () {
           const results = yield this.client.emitAsync('incoming_prepare', this.transfer)
           expect(spy).not.to.have.been.called
         })
+
+        it('should reject PSK payment without reviewPayment', function * () {
```

Comment:
I think it would be good to have a test for a PSK packet that has been tampered with (i.e. the condition doesn't match)